viewdir/listJobRoles.njk

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+{% extends 'layout-template.njk'%}
+
+{% block content %}
```

Comment:
The content is just part of the template, so `html`, `head`, and `body` should not be in this file